2.3/alpine3.7/Dockerfile

```diff
@@ -81,15 +81,15 @@ RUN set -ex \
 			| sort -u \
 			| awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
 	)" \
-	&& apk add --virtual .ruby-rundeps $runDeps \
+	&& apk add --no-cache --virtual .ruby-rundeps $runDeps \
 		bzip2 \
 		ca-certificates \
 		libffi-dev \
 		libressl-dev \
 		procps \
 		yaml-dev \
 		zlib-dev \
-	&& apk del .ruby-builddeps \
+	&& apk del --no-cache .ruby-builddeps \
```

Comment:
From https://github.com/docker-library/python/pull/330#issuecomment-413313432:

> This is actually a (recent) bug, and they're planning to fix it: https://bugs.alpinelinux.org/issues/9063
> 
> (https://git.alpinelinux.org/cgit/apk-tools/commit/?id=e1a05c74cb8ae6ab41ee960343cc64a0a9af95a5)
>
> https://git.alpinelinux.org/cgit/apk-tools/commit/?id=8a28c6d0d4f81406c67e76b2bcdbe491aa734717 appears to be the commit which introduced this "autoupdate" behavior.

On the other hand, it does affect alpine 3.7 based images (and current 3.8) since the Alpine bug is only being updated in 3.8. :cry:

Now every `apk add|del` will need to download the cache and delete it, even if it could've been done without hitting the network (and `--no-network` doesn't help).

So, I guess LGTM  cc @tianon.